docs/governance/steering/charter.md

```diff
@@ -0,0 +1,177 @@
+
+## Mission
+
+The CCC Steering Committee is the governing body of the CCC project, providing decision-making and oversight pertaining to the CCC project bylaws, sub-organizations, and financial planning.  The Steering Committee also defines the project values and structure.
+
+## Approach
+
+* Adapt the role and structure of the Steering Committee to meet the ongoing needs of the project.
+* Responsibilities not explicitly delegated to other parties<sup>[2](#footnote2)</sup> through their respective charters reside with the Steering Committee.
+* All management<sup>[1](#footnote1)</sup> responsibilities should be delegated to other parties<sup>[2](#footnote2)</sup>.
+* All technical responsibilities should be delegated to working groups. The Steering Committee shouldn't retain any technical responsibilities itself.
+* The steering committee will hold a public community call no less than once per quarter to provide updates to stakeholders regarding all CCC efforts.
+
+## Direct responsibilities of the Steering Committee
+
+The following responsibilities belong directly to the Steering Committee.
+
+* Define, evolve, and defend the non-technical vision / mission and the values of the project.
+* Delegate ownership of, responsibility for, and authority over areas of the project to specific entities<sup>[2](#footnote2)</sup>.
+* Charter and refine policy for defining new community groups<sup>[3](#footnote3)</sup>, and establish transparency and accountability policies for such groups.
+* Appoint leadership for Special Interest Groups (SIGs).
+* Define and evolve project and group<sup>[3](#footnote3)</sup> governance structures and policies<sup>[4](#footnote4)</sup>.
+* Act as a final non-technical escalation point for any CCC community group or technical effort.
+* Define and enforce requirements for community groups<sup>[3](#footnote3)</sup> to ensure ongoing positive standing, or decomission any groups not in good standing.
+* Decide who is a member of standing on the CCC project, and what privileges that entails.
+* Control and delegate access to and establish processes regarding project resources<sup>[5](#footnote5)</sup>.
+* Coordinate with FINOS regarding usage of the CCC brand and deciding which things can be called "Common Cloud Controls," as well as how that mark can be used in relation to other efforts or vendors.
+* Request funds and other support from FINOS (e.g. marketing, press, etc).
+* Interface with the FINOS Staff point of contact as needed for guidance and accountability.
+
+### FINOS Point of Contact
+
+The assigned point of contact for the steering committee to coordinate with FINOS Staff is:
+
+Rob Moffat — [@robmoffat](https://github.com/robmoffat)
+
+## Membership
+
+### Composition
+
+The Steering Committee is composed of seven (7) members and shall always be considered this size regardless of temporary vacancies, for the purpose of voting majority and supermajority considerations.
```

Comment:
Would include the length of tenure.

Reply:
This is mentioned later in the doc— the group consensus is to keep this section brief.